code/game/machinery/washing_machine.dm

```diff
@@ -239,6 +239,10 @@ GLOBAL_LIST_INIT(dye_registry, list(
 	addtimer(VARSET_CALLBACK(src, freshly_laundered, FALSE), 5 MINUTES, TIMER_UNIQUE | TIMER_OVERRIDE)
 	..()
 
+/obj/item/clothing/head/mob_holder/machine_wash(obj/machinery/washing_machine/WM)

```

Comment:
This seems redundant if you automatically release it on putting it in?

Reply:
i decided to leave this in on the off chance that someone managed to stick a held ian into the washing machine without getting "attackby" called. i can remove it if you feel it's unnecessary.

Reply:
Please do, and if you're worried about weird stuff, you can unload these in /Entered instead.